changelog/3495.feature.rst

```diff
@@ -0,0 +1,2 @@
+Made a couple of module imports lazy to reduce the import time of sunpy.map by
+~20%.
```

Comment:
On my machine, the total time improvement is now ~40% (from ~6 seconds to ~3.5 seconds).

Reply:
Go with the worst case? Or we just remove the % and say its just faster.

Reply:
Oh, I believe the "~20%" was estimated prior to the `database` improvement, so I suspect that ~40% is more accurate for everyone.

I like having a value in the changelog entry.